packages/web-components/src/label/README.md

```diff
@@ -0,0 +1,131 @@
+# Label
+
+> A label represents a caption for an item in a user interface.
+
+<br />
+
+## **Design Spec**
+
+[Link to Label Design Spec in Figma](https://www.figma.com/file/aFHbqxQlX8fetFhyYl6STb/TF---Trident-control-specs?node-id=339%3A52012)
+
+<br />
+
+## **Engineering Spec**
+
+fluent-label is a web component implementation of an HTML label element. The fluent-label several visual font sizes (small, medium, large) and font weights (regular, semibold). The fluent-label also provides appearances for required and disabled states.
+
+### Use Case
+
+Creating a simple label element with an optional info icon and optional required state
+
+<br />
+
+## Class: `Label`
+
+<br />
+
+### **Variables**
+
+<br />
+
+### **Fields**
+
+| Name       | Privacy | Type                           | Default     | Description                                                       |
+| ---------- | ------- | ------------------------------ | ----------- | ----------------------------------------------------------------- |
+| `id`       | public  | `string`                       |             | Specifies the id of label                                         |
+| `for`      | public  | `string`                       |             | Specifies the id of the form element the label should be bound to |
```

Comment:
I'm not sure this is going to work for free. The `for` attribute is a function of the HTML platform which associates a label with an input. It does not work with non-native elements (such as ARIA elements) and it may not work with form associated elements (we can test that...). 

If we follow the function of a traditional label `for` attribute in terms of what we expect to happen, we would need to use JS to find the element this associates with and dynamically add aria to it. That's pretty invasive and costly in terms of JS. Is there a better way to associate these given this consideration?